public/sass/_variables.dark.generated.scss

```diff
@@ -17,11 +17,11 @@ $theme-name: dark;
 // -------------------------
 $blue-light: #6E9FFF;
 $blue-base: #3D71D9;
-$blue-shade: rgb(90, 134, 222);
+$blue-shade: rgb(51, 96, 184);
```

Comment:
This file looks like it needs to be regenerated, now that the fusebit theme has been isolated.